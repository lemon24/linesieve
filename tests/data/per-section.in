--section '(\S+):$'
sub -s one a aa
match -s two -o '[axy]'
sub '^' '*'
---
one:
a
xb
yc
two:
a
xb
yc
