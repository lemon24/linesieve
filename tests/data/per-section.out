one
*aa
*xb
*yc
two
*a
*x
*y
